### PR TITLE
Added header component

### PR DESCRIPTION
**Pull request by sabagiorgadze**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
+    "jsx": "react",
+    "jsx": "react",
```
Comment (GeeOrrGee) [inline tsconfig.json:5]: es ratom shevcvalet?

File: package.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
   },
   "dependencies": {
     "@expo/vector-icons": "^13.0.0",
-    "@react-navigation/native": "^6.0.2",
+    "@react-navigation/native": "^6.1.9",
```
Comment (GeeOrrGee) [inline package.json:17]: ratom daainstalire es version?

File: tsconfig.json (hunk 1 above)
Comment (sabagiorgadze) [reply tsconfig.json:5]: ragac errors agdeba da quick fixma shecvala savaraudod

File: app/(homepage)/header.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { makeStyles } from "@rneui/base";
```
Comment (GeeOrrGee) [inline app/(homepage)/header.tsx:1]: es file rogorc mogwere, app/ foldershi ar unda iyos, imitom rom yvela sheqmnili file am foldershi aris avtomaturad route, rac ar gvinda. Chven mxolod reusable component gvinda, amitom reusable stuff yovetvlis sheqmeni components/ foldershi da iq chayare, da iqedan unda daaimporto sadac dagchirdeba aplikaciashi. am shemtxvevashi <Header/> component unda daaimporto da chasva app/_layout.tsx shi , root levelze navigaciis garet.

Comment (GeeOrrGee) [reply app/(homepage)/header.tsx:1]: magram erti uketesi gzac arsebobs, ristvisac componentis sheqmna saertod ar dagchirdeba da marto style unda daaamato ;) 

Hint: Check Stack navigation headerOptions

https://reactnavigation.org/docs/headers/#sharing-common-options-across-screens

**Review by GeeOrrGee — CHANGES_REQUESTED** [review]
Daamate styles da komentarebs chaxede

File: tsconfig.json (hunk 1 above)
Comment (GeeOrrGee) [reply tsconfig.json:5]: no, ase vqnat, daareverte changes da tavidan qeni yvelaferi, savaraudod homepageshi rom chasvi header magis bralia

**Pull request closed without merging**